onnxruntime/core/providers/cpu/controlflow/event_pool.cc

```diff
@@ -27,6 +23,19 @@ bool OrtEventPool::QueryEvent(int64_t id) const {
   return pool_[id].signaled.load();
 }
 
+void OrtEventPool::WaitAndResetEvent(int64_t id) {
```

Comment:
Changes in this file can be reverted. 
and this can be moved back to orttraining origin.

Reply:
done